src/widgets/paragraph.rs

```diff
@@ -202,13 +202,220 @@ impl<'a> Widget for Paragraph<'a> {
 
 #[cfg(test)]
 mod test {
+    use crate::{
+        style::Color,
+        text::{Span, Spans},
+        widgets::Borders,
+    };
+
```

Comment:
nit: can we import `use super::*;` before this line to make the formatting consistent with other modules?

Reply:
Done